spread.yaml

```
@@ -75,6 +71,23 @@ environment:
     NESTED_CUSTOM_IMAGE_URL: '$(HOST: echo "${NESTED_CUSTOM_IMAGE_URL:-}")'
     # Configure nested images to be reused on the following tests
     NESTED_CONFIGURE_IMAGES: '$(HOST: echo "${NESTED_CONFIGURE_IMAGES:-false}")'
+    # Channel used to create the nested vm
+    NESTED_CHANNEL: '$(HOST: echo "${NESTED_CHANNEL:-edge}")'
```

a more clear description would be 
```suggestion
    # Default tracking channel for snaps in the nested image, used with the ubuntu-image --channel option to create the nested vm
    NESTED_IMG_CHANNEL: '$(HOST: echo "${NESTED_IMG_CHANNEL:-edge}")'
```